### ISSUE TITLE
[WIP] Improve Accessibility of Community Events, Documents, and Tools

### ISSUE THREAD

**Issue by aleesteele** [the-turing-way/the-turing-way#2868]
### Summary

_The Turing Way_ community tools and materials need to be improved for their accessibility in a variety of contexts, building off (and learning from) the practices and ways of working being documented in #2730 and #2043 #2630  in a general audit of community tools and practices ([documented here](https://docs.google.com/spreadsheets/d/1MnvhHFQeiX6eL3IpqC1CaSiNu3tAIEDIUVaRldai5do/edit#gid=0)).

### What needs to be done?

Switching HackMD documents to Padlet for interactive check-ins. HackMDs have good readability for legacy information (can be used to archive), but cannot be used for interactive materials due to lack of compatibility with screenreaders.
- [x] Book Dash
- [ ] Coworking call 
- [ ] Collaboration Cafe 
- [ ] Demo-intro

Furthermore, a number of existing materials can be reviewed or created in order to improve the general accessibility of the community (across the many definitions of the term)
- [ ] Best practices guides for Slack, Zoom, Github (accessibility for those with disabilities, in low-bandwidth environments, across timezones, in different language) 
- [ ] Accessibility guidelines in Github repository (maybe an ACCESSIBILITY.md file?)
- [ ] Onboarding pack 

Questions:
- Should the onboarding/community event materials (like the HackMDs) be translated, to integrate the localisation practices being developed by the translation/localisation team into other social aspects of the project, such as the above?

**Comment by aleesteele** [comment]
Adding a few works in progress onboarding guides (full of typos!): 

- https://docs.google.com/document/d/1k39blCFz_uStkvRnsjPSYNKeseXFUnD9J3IYZseb_Xo/edit
- https://hackmd.io/pBysnLn2QxKMCHqPKvj9uA
- https://hackmd.io/@turingway/ttw-slack-intro

**Comment by aleesteele** [comment]
Notes from Book Dash November 2022: https://pad.sfconservancy.org/p/ttw-accessibility-notes

**Comment by EstherPlomp** [comment]
Found a resource that might be helpful: 

- [ ] https://uxdesign.cc/how-to-write-an-image-description-2f30d3bf5546 

(will add to this if I find more resources!)

**Comment by aleesteele** [comment]
Closing this as tied to our #3581 :)
